frontend/public/locales/en/common.json

```
@@ -208,7 +209,7 @@
   "account-deletion-token-mismatch": "The account deletion request failed to authenticate.",
   "privacy-policy": "Privacy Policy",
   "privacy-policy-description": "By using this website, you agree to the Privacy Policy.",
-  "terms-and-conditions": "Terms and conditions",
+  "terms-and-conditions": "Terms And Conditions",
```

Traditionally minor words/short conjunctions are not capitalized for title case

```suggestion
  "terms-and-conditions": "Terms and Conditions",
```